src/components/mgt-person/mgt-person.ts

```diff
@@ -463,6 +463,7 @@ export class MgtPerson extends MgtTemplatedComponent {
    */
   protected async loadState() {
     const provider = Providers.globalProvider;
+    const graph = provider.graph.forComponent(this);
```

Comment:
this would throw an exception if provider is null. Move this line above the check for `if (this.personDetails)`

Reply:
Oops.. you are right